src/main/java/com/diffblue/javademo/tictactoe/TicTacToeGame.java

```diff
@@ -0,0 +1,101 @@
+package com.diffblue.javademo.tictactoe;
+
+
+public class TicTacToeGame {
+
+  private Board board = new Board(3);
+
+
+  /**
+   * Check move of players.
```

Comment:
This description doesn't seem to match the code. It suggests that a move is being entered and that move is being checked for validity. The code is checking to see if the board is valid.

Reply:
Updated the descirption